static/web.css

```diff
@@ -533,11 +533,31 @@ button[disabled]:last-child::before {
 :root .ace-github .ace_marker-layer .ace_selected-word { background: rgba(245, 245, 255, 0.5); }
 :root .ace-github .ace_print-margin { background: rgba(209, 209, 209, 0.5); }
 
-/* rustc output. bright versions of red, yellow, green, cyan and magenta are used, but black, blue and gray could theoretically be added with dim versions of them. */
-.rustc-output {
+/* ponyc output. bright versions of red, yellow, green, cyan and magenta are used, but black, blue and gray could theoretically be added with dim versions of them. */
+pre.output {
```

Comment:
Since I see that you changed other instances of `.rustc-output` for `.ponyc-output`, is `pre.output` intentional, or do you wanted to use `.ponyc-output` here?

Reply:
The `.ponyc-output` class is for compilation output, while the stderr and stdout output tags have a different class. The css class `output` is applied to all of those output sections, so `pre.output` applies to all of them, while `.ponyc-output` only applies to the compiler output. It is being special as it always comes out at the top.

Reply:
See https://github.com/ponylang/pony-playground/pull/4/files/0f5e9a7f1237420a37905a2c891ced5249511d34#diff-e6423f3beb4a6d65487cdb514829c863aa2fd1862d11bf452582d1052e771d52R153